extensions/cdc-debezium/src/main/java/com/hazelcast/jet/cdc/ChangeRecord.java

```diff
@@ -21,27 +21,34 @@
 import javax.annotation.Nonnull;
 
 /**
- * Information pertaining to a single data change event (insertion,
- * delete or update), affecting a single database record.
+ * Information pertaining to a single data change event (insertion, delete or
```

Comment:
why not "insert, delete or update" (alternatively, "insertion, deletion or updating")